### ISSUE TITLE
partial name collision

### ISSUE THREAD

**Issue by straandr** [Atomic-Reactor/Toolkit#5]
Creating materials with same base name causes assembler to crash. Is this an issue with CLI? Toolkit? or Assembler? Can we register the partials with names qualified by their parent directory or something?

**Comment by camdagr8** [comment]
This is an issue with handlebars actually. You can only declare a partial once.

**Comment by straandr** [comment]
Yes, but my point is that it shouldn't crash. If you're running the server and accidentally cause this issue, the process should report an error, and if it's watching for changes, it should recover when the issue is resolved.
